lib/profile/index.js

```diff
@@ -34,8 +34,8 @@ function defaults ( ) {
     // WARNING: use SMB with caution: it can and will automatically bolus up to max_iob worth of extra insulin
     , enableUAM: true // enable detection of unannounced meal carb absorption
```

Comment:
`enableUAM` is defaulted to `false` in Trio, so maybe it's worth updating that in this PR as well.

Reply:
Thanks for your review, the changes has been made.

Reply:
Actually, seeing that UAM has been on by default in openaps/oref0 since 2017, we might want to reconsider setting `enableUAM` to `false`. But in that case perhaps we should consider changing this in Trio defaults as well.

This is not an important matter for me though, and we can keep it as is too.

Reply:
I agree that UAM should be enabled by default. Not sure if that’d cause everyone’s settings to get reset again or not, though. I think there’s a way to do it without causing a reset, but the code wouldn’t be as clean. Maybe when settings are moved out of json it’ll be easier to update defaults?